### DISCUSSION TITLE
Surace Pro 1 - Cannot enter the MOK Blue Screen

### DISCUSSION THREAD

**Discussion by chrisjeanson** [linux-surface/linux-surface#1570] [Q&A]
Hey there, I installed Fedora 40 Workstation on a Surface Pro 1 and thought to try linux-surface so I can enable secure boot. Following the instructions, after I install surface-secureboot I am told to reboot where I can enter the password to enroll the key. However, upon rebooting I notice that the Surface doesn't send any power to the single USB port on the tablet so I cannot use my keyboard that is plugged directly into the tablet to get past the "press any key" timer.

I also tried using a usb splitter to see if that would help, but once I tried another keyboard that has a backlight I noticed right away that it isn't receiving any power until the blue screen prompt disappears and the regular boot process continues.

Is there another way to accomplish this step?

Thanks!

**Comment by jifargithub** [comment]
I also encountered the same problem and look forward to resolving this bug
